web/static/sass/wtt.scss

```diff
@@ -299,6 +299,32 @@ textarea {
     }
 }
 
+.hero--responsiveness {
+    background-image: url(../img/heroes/index-responsiveness.jpg);
```

Comment:
Not sure if you already tried, but would a smaller picture at smaller widths save some Kb?